scripts/components/LviAddReviewSection.ts

```diff
@@ -0,0 +1,39 @@
+import LviAddReviewSectionDesign from 'generated/my-components/LviAddReviewSection';
+import { themeService } from 'theme';
+
+const { height } = themeService.getStyle('.lviAddReviewSection');
+
+export default class LviAddReviewSection extends LviAddReviewSectionDesign {
+    pageName?: string | undefined;
+    constructor(props?: any, pageName?: string) {
+        super(props);
+        this.pageName = pageName;
+    }
+    static getHeight(): number {
+        return height;
+    }
+    get image(): string {
+        return this.flAddReviewSection.image;
+    }
+    set image(value: string) {
+        this.flAddReviewSection.image = value;
+    }
+    get review(): string {
+        return this.flAddReviewSection.review;
+    }
+    set review(value: string) {
+        this.flAddReviewSection.review = value;
+    }
+    get addReviewText(): string {
+        return this.flAddReviewSection.addReviewText;
+    }
+    set addReviewText(value: string) {
+        this.flAddReviewSection.addReviewText = value;
```

Comment:
If this is a field for "Add review" we should assign this lang text on constructor and delete this setter. If its changeble text field then its name is wrong.

Reply:
Yeah, it's true. Fixed.